### PR TITLE
Adds a handful of nice sprite related things from TG. (Inhands etc.)

### PR DESCRIPTION
**Pull request by Septoxide**
## About The Pull Request

Pretty much ports all these:
https://github.com/tgstation/tgstation/pull/50962 (Singularity hammer) (Only the sprites)
https://github.com/tgstation/tgstation/pull/57089 (Telescopic baton sprite) (I modified it slightly)
https://github.com/tgstation/tgstation/pull/46975 (Animated flamethrower inhand, and it makes light and sound on activation)
https://github.com/tgstation/tgstation/pull/52736 (syringes visible in syringe guns) (I fixed the DNA gun's reload sound and icon)
https://github.com/tgstation/tgstation/pull/45581 (Inhand sprites for bedsheets) (I added one for the magicians cape)
https://github.com/tgstation/tgstation/pull/47549 (Inhand sprites for the airlock painter) (I fixed the right hand one)
https://github.com/tgstation/tgstation/pull/46975 (Inhand sprites for wood planks)
https://github.com/tgstation/tgstation/pull/53291 (Inhand sprite for the advanced extinguisher) (I added a grey band to it)

## Why It's Good For The Game

The singularity hammer has been using the default mjolnir sprite as a placeholder forever, and this one looks amazing:

ndKFWTfigG

<br>
Additionally the old telescopic baton looks really bad, TG recently got a new one that is much more visible in your inventory, I modified it a little which resulted in this:

comparison

<br>
I also saw a PR that added flamethrowers making light, so I am porting that, plus sound effects and an animated inhand sprite, which makes flamethrowers look more interesting:

https://user-images.githubusercontent.com/79375148/110732538-4435be00-821c-11eb-95e8-2aa9ce408395.mp4

<br>
Syringe guns have reloading noises, and syringes can be seen inside of them, which is both satisfying and useful:

https://user-images.githubusercontent.com/79375148/110732360-f5882400-821b-11eb-8597-573bb4834147.mp4

<br>
Last is inhand sprites for bedsheets, wood planks, advanced extinguishers, and the airlock painter. Which are just nice to have:

https://user-images.githubusercontent.com/79375148/110756738-79ed9d80-8242-11eb-98c3-1330d9babb2d.mp4

## Changelog
:cl: Hollandaise, Nebulacrity, LemonInTheDark, Flareguy, smallveggiepaws, Mothblocks, Twaticus, uomo91
tweak: You can now see if syringe guns are loaded just by looking at them.
tweak: Flamethrowers produce light.
soundadd: Flamethrowers play a sound when activated, syringe guns also make sounds when reloaded.
imageadd: Flamethrowers now have an animated inhand sprite.
imageadd: Wood planks, bedsheets, advanced extinguishers, and airlock painters now have inhand sprites.
imageadd: New telescopic baton and singularity hammer sprites.
imagedel: Old telebaton sprites.
/:cl:

### CODE REVIEW

**Comment by NickEmbased** [comment]
Awesome

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 1]
```diff
@@ -10,10 +10,13 @@
 	materials = list(/datum/material/iron=2000)
 	clumsy_check = 0
 	fire_sound = 'sound/items/syringeproj.ogg'
+	var/load_sound = 'sound/weapons/shotguninsert.ogg'
 	var/list/syringes = list()
 	var/max_syringes = 1
+	var/has_syringe_overlay = TRUE ///If it has an overlay for inserted syringes. If true, the overlay is determined by the number of syringes inserted into it.
 
 /obj/item/gun/syringe/Initialize()
+	update_icon()
```
Comment (PowerfulBacon) [inline code/modules/projectiles/guns/misc/syringe_gun.dm]: update_icon should be after init

File: code/modules/projectiles/guns/misc/syringe_gun.dm [hunk 2]
```diff
@@ -33,7 +36,7 @@
 /obj/item/gun/syringe/process_chamber()
 	if(chambered && !chambered.BB) //we just fired
 		recharge_newshot()
-
+	update_icon()
```
Comment (PowerfulBacon) [inline code/modules/projectiles/guns/misc/syringe_gun.dm:39]: add a new line after this between the new proc

Comment (Septoxide) [reply code/modules/projectiles/guns/misc/syringe_gun.dm:39]: alrighty I added it

File: code/modules/projectiles/guns/misc/syringe_gun.dm (hunk 1 above)
Comment (Septoxide) [reply code/modules/projectiles/guns/misc/syringe_gun.dm]: done that too (I think)

File: code/game/objects/structures/bedsheet_bin.dm [hunk 3]
```diff
@@ -141,105 +156,121 @@ LINEN BINS
...
 /obj/item/bedsheet/wiz
 	name = "wizard's bedsheet"
 	desc = "A special fabric enchanted with magic so you can have an enchanted night. It even glows!"
 	icon_state = "sheetwiz"
+	item_state = "sheetwiz"
 	item_color = "wiz"
 	dream_messages = list("a book", "an explosion", "lightning", "a staff", "a skeleton", "a robe", "magic")
 
 /obj/item/bedsheet/nanotrasen
-	name = "\improper Nanotrasen bedsheet"
```
Comment (francinum) [inline code/game/objects/structures/bedsheet_bin.dm:228]: Why did you remove this exactly?

Comment (Septoxide) [reply code/game/objects/structures/bedsheet_bin.dm:228]: Well the PR it was porting stuff from had no "\improper" in it, so I just assumed it was a mistake, but someone told me it actually has a function so I will put it back.

Comment (francinum) [reply code/game/objects/structures/bedsheet_bin.dm:228]: It's a byond text macro that pretty much forces improper nouns, without it and with capitalization byond assumes it's a proper noun,

`a Nanotrasen bedsheet` vs `the Nanotrasen bedsheet`

**Review by PowerfulBacon — APPROVED** [review]
